artemis-distribution/pom.xml

```diff
@@ -117,16 +117,6 @@
          <artifactId>artemis-console</artifactId>
          <type>war</type>
       </dependency>
-      <dependency>
-         <groupId>org.apache.activemq</groupId>
-         <artifactId>activemq-branding</artifactId>
-         <type>war</type>
-      </dependency>
-      <dependency>
-         <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-plugin</artifactId>
-         <type>war</type>
-      </dependency>
```

Comment:
This is as much a comment on the unaltered bit remaining, as the removals.

The binary distribution assembly has a number of entries in its LICENCE file (and files it references) to do with covering the console bits. Those files haven't been changed here, but it seems almost certain they will need to be. That is, `artemis-distribution/src/main/resources/licenses/bin/LICENSE` and files under dir `artemis-distribution/src/main/resources/licenses/bin/licenses/`.